pureelk.sh

```diff
@@ -175,9 +179,31 @@ start_containers() {
   else
       print_warn "$PUREELK is already running."
   fi
+  
+  print_info "Start Logrotate container..."
+  RUNNING="$(docker inspect -f '{{.State.Running}}' $LOGROTATE)"
+  if [ $? -eq 1 ];
+  then
+      print_warn "$LOGROTATE doesn't exist, starting..."
+      docker run -d -v /var/lib/docker/containers:/var/lib/docker/containers -v /var/log/pureelk:/var/log/pureelk --name $LOGROTATE \
+        -e "LOGS_DIRECTORIES=/var/lib/docker/containers /var/log/pureelk" \
```

Comment:
Should you also set LOGROTATE_COPIES to a fix number? Otherwise wouldn't we still keep the log files forever?